ref/Microsoft.Build/net/Microsoft.Build.cs

```diff
@@ -1050,7 +1050,7 @@ public partial class BuildSubmission
     public partial class HostServices
     {
         public HostServices() { }
-        public Microsoft.Build.Framework.ITaskHost GetHostObject(string projectFile, string targetName, string taskName) { throw null; }
+        public System.Threading.Tasks.Task<Microsoft.Build.Framework.ITaskHost> GetHostObject(string projectFile, string targetName, string taskName) { throw null; }
```

Comment:
Isn't this a breaking change for VS? My instinct is to hide all of this and call `Thread.Join()` to wait on the result of the remote thread. Is that feasible?